plugins/cilium-cni/cmd/cmd.go

```diff
@@ -396,6 +396,26 @@ func setupLogging(n *types.NetConf) error {
 	return nil
 }
 
+func reserveLocalIPPorts(conf *models.DaemonConfigurationStatus, sysctl sysctl.Sysctl) error {
+	if conf.IPLocalReservedPorts == "" {
+		return nil
+	}
+
+	// Note: This setting applies to IPv4 and IPv6
+	const param = "net.ipv4.ip_local_reserved_ports"
+	var value = conf.IPLocalReservedPorts
+
+	// Append our reserved ports to the ones which might already be reserved
+	existing, err := sysctl.Read(param)
+	if err != nil {
+		return err
+	}
+	if existing != "" {
```

Comment:
I'm assuming `CNI ADD` should only ever be called once so this might be fine, but if this could be used after initial setup then it might be necessary to check what the existing values are to avoid re-adding duplicate values later.

Reply:
Good call. I didn't know it when I wrote that code, but during testing I noticed that the kernel will actually automatically remove duplicates for us. So append only should work just fine. 

I'll add a comment that we're offloading deduplication to the kernel :sloth: